src/helpers/url.ts

```diff
@@ -44,4 +44,4 @@ export const isValidUrl = (str: string) => {
   return !!(str.match(urlRegex) || str.match(mailtoUrlRegex) || str.match(telUrlRegex) || str.match(smsUrlRegex));
 };
 
-export const isValidImageUrl = (str: string) => isValidUrl(str) && !!str.match(/\.(gif|jpe?g|tiff?|png|webp|bmp)$/i);
+export const imageContentTypes = ['binary/octet-stream', 'image/jpeg', 'image/png', 'image/gif'];
```

Comment:
We have to support webp, tiff, bmp and jpg as well.

Reply:
These are the valid image mime types as per chatGPT: https://chat.openai.com/share/d44b7044-8482-471e-94f1-fe5d2d4b6407